src/harness/rwcRunner.ts

```diff
@@ -96,9 +105,6 @@ module RWC {
                                 if (useCustomLibraryFile) {
                                     inputFiles.push(getHarnessCompilerInputUnit(fileRead.path));
                                 }
-                                else {
-                                    inputFiles.push(Harness.getDefaultLibraryFile());

```

Comment:
This fails because Harness.getDefaultLibraryFile will call into ts.sys (which is swap to use data Playback in runWithIOLog) that looks into log array (i,e array representation of the input json) so it will fail to find default lib.d.ts